L1Trigger/VertexFinder/python/VertexNTupler_cff.py

```diff
@@ -1,15 +1,14 @@
 import FWCore.ParameterSet.Config as cms
-from L1Trigger.VertexFinder.VertexProducer_cff import VertexProducer
 
-L1TVertexNTupler = cms.EDAnalyzer('VertexNTupler',
+l1tVertexNTupler = cms.EDAnalyzer('VertexNTupler',
   inputDataInputTag = cms.InputTag("InputDataProducer","InputData"),
   genParticleInputTag = cms.InputTag("genParticles",""),
-  l1TracksInputTags    = cms.VInputTag( VertexProducer.l1TracksInputTag ),
+  l1TracksInputTags    = cms.VInputTag( l1tVertexProducer.l1TracksInputTag ),
```

Comment:
The `import` was removed, so here the object `l1tVertexProducer` does not exist, should be replaced with the explicit label (string) of the InputTag. Same below for `VertexReconstruction = l1tVertexProducer.VertexReconstruction,`. Also, this file should be a `cfi`, not a `cff`.

Reply:
Done